### DISCUSSION TITLE
Some Basic Things I would really like to be added

### DISCUSSION THREAD

**Discussion by Sayak-W** [pear-devs/pear-desktop#1352] [Ideas]
1. The album color theme plugin doesn't entirely change the theme, the whole library portion stays black. So it'd be great if the whole page's color changes.
2. I can't turn off the top bar menu. It'd be great if I could turn it off.
Some Improvements-
1. The genius lyrics doesn't work properly. And since I mostly listen to j-pop it doesn't give me the romanized lyrics every time even though I've turned it on.
2. I know the crossfade plugin is in beta, so problems I face are: when the next song is played the audio is totally cut off, I have to tap on volume control, then it comes back.
All other things are great, I really appreciate you doing this. Keep up the great work.

**Comment by inson1** [comment]
"The album color theme plugin doesn't entirely change the theme, the whole library portion stays black. So it'd be great if the whole page's color changes." yes please + search + album page (color of that abum)
